content/en/docs/tasks/administer-cluster/dns-debugging-resolution.md

```diff
@@ -110,9 +110,9 @@ have to deploy it manually.
 Use `kubectl logs` command to see logs for the DNS daemons.
 
 ```shell
-$ kubectl logs --namespace=kube-system $(kubectl get pods --namespace=kube-system -l k8s-app=kube-dns -o name) -c kubedns
-$ kubectl logs --namespace=kube-system $(kubectl get pods --namespace=kube-system -l k8s-app=kube-dns -o name) -c dnsmasq
-$ kubectl logs --namespace=kube-system $(kubectl get pods --namespace=kube-system -l k8s-app=kube-dns -o name) -c sidecar
+$ kubectl logs --namespace=kube-system $(kubectl get pods --namespace=kube-system -l k8s-app=kube-dns -o name | head -1) -c kubedns
```

Comment:
Please remove the initial prompts, to follow the style guide. This probably means that you need to add whitespace after each command for readability.

/hold